src/theme.js

```diff
@@ -1,21 +1,29 @@
 export const grommetToolsTheme = {
-    global: {
-      colors: {
-        'DesignerYellow': '#FDEF74',
-        'ThemerOrange': '#F9D36E',
-        'ImagerOrange': '#F4AD68',
-        'PublisherPink': '#FFD8D8',
-        'SlidesBlue': '#6AE4FB',
-        'TabularGreen': '#9BFAD8',
-        'noCodeBackground': 'rgba(0,0,0,0.06)',
-        'darkGrey': '#666666',
-        background: {
-          light: '#ffffff',
-          dark: '#000000',
-        },
+  global: {
+    edgeSize: {
+      marginXL: '200px',
+    },
+    size: {
+      middle: '500px',
```

Comment:
What is this used for? we are normally trying to avoid random numbers.

Reply:
It is used for the size of the Grommet Tool Suite cards. I can change the name to something more descriptive to make it more clear